tflint/annotation.go

```diff
@@ -66,6 +76,64 @@ func NewAnnotations(path string, file *hcl.File) (Annotations, hcl.Diagnostics)
 	return ret, diags
 }
 
+// jsonAnnotations finds annotations in .tf.json files. Currently only
+// file-level ignores are supported, by specifying a root-level comment
+// property (with key "//") which is an object containing a string property
+// with the key "tflint-ignore-file".
+func jsonAnnotations(path string, file *hcl.File) (Annotations, hcl.Diagnostics) {
+	ret := Annotations{}
+	diags := hcl.Diagnostics{}
+
+	var data map[string]json.RawMessage
+	if err := json.Unmarshal(file.Bytes, &data); err != nil {
+		diags = append(diags, &hcl.Diagnostic{
+			Severity: hcl.DiagError,
+			Summary:  "failed to unmarshal file as JSON",
+			Detail:   err.Error(),
+			Subject: &hcl.Range{
+				Filename: path,
+				// Unfortunately Start and End are difficult to determine here
+				// since encoding/json does not expose source metadata.
+			},
```

Comment:
```suggestion
			Subject: &hcl.Range{
				// Cannot set Start/End because encoding/json does not expose it
				Filename: path,
			},
```

Avoid feelings language (unfortunately, difficult), emphasize constraints and outcome